cvc/certificates.py

```diff
@@ -38,6 +38,20 @@ def decode(self, data):
         self.__a = ASN1().decode(self.__data)
         return self
 
+    def decode_authorization_bits(self):
```

Comment:
Once CHAT bytes are exposed via chat(), I am not sure the purpose of exposing them as bits. If it for just printing them, it should be placed in the printing script and not in the API.

Reply:
I'm with you on that and moved the function over to cvc_print.py